### ISSUE TITLE
Cookie not in this host's domain

### ISSUE THREAD

**Issue by smootok** [3846masa/axios-cookiejar-support#285]
I'm trying to store instagram cookies
```js
const axios = require('axios').default
const axiosCookieJarSupport = require('axios-cookiejar-support').default
const tough = require('tough-cookie')

axiosCookieJarSupport(axios)
const cookieJar = new tough.CookieJar()
axios.defaults.jar = cookieJar
axios.defaults.withCredentials = true

exports.getProfilePic = async (req, res, next) => {
  const baseUrl = 'https://www.instagram.com/'
  const response = await axios.get(baseUrl)
  console.log(response.config.jar.toJSON())
  res.json(cookieJar)
}
```
I get this error

`Error: Cookie not in this host's domain. Cookie:i.instagram.com Request:www.instagram.com`

 The base URL is `https://www.instagram.com/` and I get some cookies from`i.instagram.com`

 Is it possible to store subdomains cookies?.

**Comment by moltar** [comment]
Was getting the same when using axios version `0.19.2`, downgrading back to `0.19.0` fixed the problem.

**Comment by moltar** [comment]
Getting the same error on `1.0.0`

**Comment by moltar** [comment]
Not sure why I said `0.19.0` in my previous message, this isn't even a published version.

I meant:

```
"axios-cookiejar-support": "0.5.0",
```

**Comment by Stevenje** [comment]
@moltar the issue happens with axios 19+ on the main page there is a section for how to handle this although I keep getting the same errors as you and need to update to later axios versions due to bug fixes.

**Comment by Stevenje** [comment]
@moltar simple option which got everything working with version 1 of axios-cookiejar-support and axios 20 was to not use axios.create and just stick to using axios.get etc

**Comment by 3846masa** [comment]
Sorry for the delay in response.
This issue has been resolved with the release of v2.0.0.

If you think this issue is still unresolved, please reopen it.
